### ISSUE TITLE
Move class-parameters from individual fixed assets to asset classes

### ISSUE THREAD

**Issue by ehuelsmann** [ledgersmb/LedgerSMB#3605]
Move these class properties to the class:

* Usable life
* Asset account
* Depreciation account
* Expense account

Note that `Asset account` and `Depreciation account` already are on the class definition (but it's unclear which ones are being used).  'Expense account` hasn't been defined on the asset class though (but should).

**Comment by aungzwin10** [comment]
I agree, it will be better when Expense Account define in class. But for Usable Life, each asset will have different Usable Life although they have same Class. Let's say, Office Equipment like furniture, table, chair but each table and chair may have different usable life.

**Comment by ehuelsmann** [comment]
> I agree, it will be better when Expense Account define in class. But for Usable Life, each asset will have different Usable Life although they have same Class. Let's say, Office Equipment like furniture, table, chair but each table and chair may have different usable life.

Can you explain why each chair has a different useable life? Here, all laptops have a standard (economic) useable life. Same for desks and chairs. The fact that they are of different quality is not a criterion here ("here" as in "where I live"; not referring to the issue itself).

**Comment by aungzwin10** [comment]
We misunderstood something. What I understand about the Asset Class is: we register class as more generic.
E.g. In the Asset Class `Office Equipment`, there are multiple assets including chair, table, other furniture.
You may register asset class as `Chair`, `Table`, `Laptop`. right?

**Comment by ehuelsmann** [comment]
Yes, you can have classes for chairs, tables and laptops separated. It is indeed how i would set it up. Multiple classes (chairs, tables and laptops) can be linked to the Office Equipment asset account. That way, the sum of the classes equals the account balance.
